core/src/main/java/com/amplitude/core/platform/intercept/IdentifyInterceptFileStorageHandler.kt

```diff
@@ -0,0 +1,118 @@
+package com.amplitude.core.platform.intercept
+
+import com.amplitude.common.Logger
+import com.amplitude.core.events.BaseEvent
+import com.amplitude.core.events.IdentifyOperation
+import com.amplitude.core.utilities.FileStorage
+import com.amplitude.core.utilities.toEvents
+import kotlinx.coroutines.CoroutineDispatcher
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.launch
+import org.json.JSONArray
+import org.json.JSONException
+
+class IdentifyInterceptFileStorageHandler(
```

Comment:
Do we need dedicated Storage handlers? Could we use an existing event storage class and handle the all the merge logic in IdentifyInterceptor instead?

Reply:
yeah, that's my first attempt. Break it down mostly on how we handle the events remove. We could discuss later about this.